resources/js/Shared/Select.vue

```diff
@@ -28,7 +28,7 @@
               :label="customLabelKey"
               :data-cy="datacy"
               :close-on-select="true"
-              @input="broadcast(selected)"
+              @input="broadcast"
```

Comment:
are you sure here? how does it know what's selected?

Reply:
The value is sent in the event's parameter, to the `broadcast` function.

Reply:
See https://vue-select.org/guide/values.html#getting-and-setting